app/services/google_service/client_service.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module GoogleService
+  class ClientService
+    require 'httparty'
+    require 'colorize'
+
+    USER_AGENT = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) '\
+                'AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.77 Safari/537.36'
+
+    def initialize(keyword, lang = 'en')
+      escaped_keyword = CGI.escape(keyword)
+      @uri = URI("https://google.com/search?q=#{escaped_keyword}&hl=#{lang}&gl=#{lang}")
```

Comment:
How about defining the base search url as constant like USER_AGENT?
```
BASE_SEARCH_URL = `https://google.com/search`
```
```suggestion
      @uri = URI("#{BASE_SEARCH_URL}?q=#{escaped_keyword}&hl=#{lang}&gl=#{lang}")
```

I think its a bit cleaner

Reply:
Can do! ;-)